extensions/resource-deployment/src/ui/radioGroupLoadingComponentBuilder.ts

```diff
@@ -30,14 +30,15 @@ export class RadioGroupLoadingComponentBuilder implements azdata.ComponentBuilde
 	}
 
 	async loadOptions(optionsInfo: OptionsInfo | (() => Promise<OptionsInfo>)): Promise<void> {
-		if (typeof optionsInfo !== 'object') {
-			optionsInfo = await optionsInfo();
-		}
 		this.component().loading = true;
 		this._optionsDivContainer.clearItems();
-		let options: (string[] | azdata.CategoryValue[]) = optionsInfo.values!;
-		let defaultValue: string = optionsInfo.defaultValue!;
 		try {
+			if (typeof optionsInfo !== 'object') {
```

Comment:
What's this change for?

Reply:
So that when an exception is thrown while calling optionsInfo() the catch block handles displaying it correctly in the divContainer.

Reply:
Please include a screenshot/gif of what this looks like now

Reply:
This is what it looks like after the fix: 
![image](https://user-images.githubusercontent.com/41588310/88562572-b2a77200-cfe5-11ea-9f00-5000e93a6c8a.png)

This is what it did before the fix. The spinner next to the 'Cluster Context:' label would just keep spinning forever indicating load which would never complete because there is no file.
![image](https://user-images.githubusercontent.com/41588310/88562974-3c573f80-cfe6-11ea-8ed4-0df19e8c9156.png)

Reply:
And I assume you've tested it that it clears the error message correctly when you then select a valid kube config?

Reply:
yup. With valid kube config file, the new radio buttons get shown. That reminds me to test with a an invalid file that is picked from file picker that exists but is not parseable as a kube config file. Will test with that before check-in as well.

Reply:
@ranasaria I do see an accessibility issue here, we should leverage the wizard error banner to show errors, otherwise it won't get announced unless you make this a live region.